### PR TITLE
Tweak the clock app's settings.

### PR DESCRIPTION
**Pull request by nicomiguelino**
### Checklist

- [x] Change text-type settings to string.
- [x] Change confidential settings to secret.
- [x] Replace `sentry_id` and `ga_api_key` with `tag_manager_id` instead.

### CODE REVIEW

**Comment by nicomiguelino** [comment]
@vpetersson / @sergey-borovkov / @korvyashka
Given the following changes &mdash; Screenly/Screenly#343 &mdash; I decided to create this PR so that `string` will be used instead of `text`. I also changed the types of API keys (and similar settings) to `secret`.

**Comment by vpetersson** [comment]
Why do we need Google Analytics API key for clock?

**Comment by nicomiguelino** [comment]
@vpetersson, I just replaced the GA ant Sentry settings with Google Tag Manager ID instead. Requesting another review... Thanks!

File: edge-apps/clock/screenly.yml [hunk 1]
```diff
@@ -8,23 +8,20 @@ author: Screenly, Inc.
...
-    type: text
+    type: string
     default_value: 'false'
+    title: disable_analytics
     optional: true
     help_text: Whether to disable Sentry and Google Analytics integrations
-  ga_api_key:
-    type: text
+  tag_manager_id:
+    type: secret
```
Comment (vpetersson) [inline edge-apps/clock/screenly.yml:17]: This should be text.

**Pull request merged**